packages/shared/package.json

```diff
@@ -13,6 +13,7 @@
   "sideEffects": false,
   "main": "src/index.ts",
   "scripts": {
+    "build": "run-p build:*",
```

Comment:
`pnpm packages` was not building the shared types, since Rollup was not configured to look for them until now.

Note that since `@h5web/shared` is a dependency of the other packages, pnpm knows to call its `build` script first, so the other packages' builds are guaranteed to find the shared built types under `packages/shared/dist-ts`.